packages/web-react/scripts/prepareDist.js

```diff
@@ -0,0 +1,79 @@
+// The Spirit Web React source that is published to npm is located in the
```

Comment:
This script with aid of `entryPoints.js` just add simple `package.json` to every directory to make `components` or `components/Button` accessible via import.